python/python/glide/async_commands/core.py

```diff
@@ -136,6 +136,18 @@ class UpdateOptions(Enum):
     GREATER_THAN = "GT"
 
 
+class SortOrder(Enum):
+    """
+    SORT order options: options for sorting elements.
+
+    - ASC: Sort in ascending order.
+    - DESC: Sort in descending order.
```

Comment:
> What is CME/CMD?

_CMD_: Cluster Mode Disable.
_CME_: Cluster Mode Enable.

As I wrote in the PR itself:

> In CME, two arguments (BY/GET) aren't supported, so we dicided to create separate APIs for CMD and CME so in CME those 2 two won't appear as optional.

**So for your questions:**
> Why do you split cluster and standalone implementations?

Because we want to limit this API call so the customer won't be able to send unsupported args to this function, so for ClusterClient we eliminated those BT\GET function's args.

> Why do you not include BY and GET pattern options for standalone?

I do for standalone client, but not for ClusterClient, just for the reason I described. 

> You should also allow specifying route for cluster commands and document it.

This command operate on a specific key which like any other commands it doesn't need to get the route as an argument.

Reply:
Oh I see. The definition of CMD/CME answers to all of my questions.
Please, don't use it - it is not a Redis term. PR should be understandable for everyone.
Please, update PR description, and/or code comments. Add new function signatures to PR description for better explanation. I'll do another review round.